web-frontend/src/i18n/en_US.json

```diff
@@ -93,5 +93,9 @@
   "receive": {
     "account-receive": "Generate a link",
     "account-address": "Account address: "
+  },
+  "placeholder": {
+    "placeholder-banner": "Our new feature is coming soon",
+    "placeholder-message": "If you're here, it's because you wanted to see your transactions! Don't panic, you'll soon be able to do in 3 months."
```

Comment:
`placeholder.placeholder-message` is too generic

we will have multiple placeholder pages:
- teaser transaction page
- teaser asset page
- teaser contact page

Reply:
@Thykof i'll work on the messages it should be quick